localnet/with-ibc/relayer/template/run_relayer.sh

```diff
@@ -5,6 +5,9 @@ command -v shellcheck >/dev/null && shellcheck "$0"
 
 file_path=".relayer/config/config.yaml"
 
+echo "wait until chains up"
+sleep 10
```

Comment:
How about checking if port is open?

Reply:
I tried it but failed because relayer image does not have timeout command.
I got `timeout command not found` error.